src/components/itemContextMenu.js

```diff
@@ -11,6 +11,20 @@ import ServerConnections from './ServerConnections';
 import toast from './toast/toast';
 import * as userSettings from '../scripts/settings/userSettings';
 
+function getDeleteCommandName(type) {
+    switch (type) {
```

Comment:
in a file littered with if/else's, i thank you for using a switch lol

Reply:
:D